README.md

```diff
@@ -102,6 +103,8 @@ Add the libname to either requirements.in or dev-requirents.in, then either upgr
 
 ### Cleaning example code
 Before start creating your own apps, run the command `make clean_examples` in order to clean up the example apps from the front and backend.
+Deregister the example app by removing `'exampleapp.apps.ExampleappConfig'` from ``{{project_name}}/settings/base.py`` and adjust 
```

Comment:
This step would be better to be done before the command `make clean_examples`. Just to avoid getting errors after removing the code. Please also mention that the example app URLs  should be removed from ``{{project_name}}/urls.py``

Reply:
done